src/fable/Fable.Compiler/FSharp2Fable.Util.fs

```diff
@@ -540,7 +540,8 @@ module Types =
         let makeProperties (tdef: FSharpEntity) =
             tdef.MembersFunctionsAndValues
             |> Seq.choose (fun x ->
-                if x.IsPropertyGetterMethod
+                // TODO: Remove anymore unneeded props?
+                if x.IsProperty
```

Comment:
I'm not sure about the duplicates but the property used for the test doesn't seem to work if you ```IsPropertyGetterMethod```

```
type PropertyJson() =
    member val Prop1 = {a="";b=0} with get,set
```

Also the schema (and test) with just ```IsProperty``` for the following code seems ok
```
type PropertyJson2() =
    let mutable prop1 = {a=""; b=0}
    member this.Prop1 with get () = prop1 and set value = prop1 <- value
```

schema:
```
    var result = _fableCore.Serialize.ofJson(json, 1, [{
      $type: "Fable.Tests.Json.Child"
    }, {
      $type: "Fable.Tests.Json.PropertyJson2",
      Prop1: 0
    }]);
```